### ISSUE TITLE
ProductAlert: Product alerts not showing in admin side product edit page

### ISSUE THREAD

**Issue by chiragp-wagento** [magento/magento2#10007]
### Preconditions
Below is environment I am using

1. Magento version CE 2.1.5
2. PHP Version 7.0.15
3. MySQL Version 5.7.18

### Steps to reproduce

1. Make any product out of stock and view the product in front side
2. Make sure you are logged in and click on " Notify me when this product is in stock " link to subscribe for stock alert and it will show "Alert subscription has been saved." success message.
3. Now, when we edit the same product in admin side then it should show Product Alert grid with 2 section, 1. Price Alert Subscriptions and 2. Stock Alert Subscriptions

### Expected result
1. It should show Product Alert grid in product edit page in admin side

### Actual result
1. Product alert grids are not showing in product edit page

After troubleshoot I have found that below code is not added in catalog_product_edit.xml that's why tab is not showing.

```
<referenceBlock name="product_form">
            <block class="Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Alerts" name="alert.tab">
                <arguments>
                    <argument name="config" xsi:type="array">
                        <item name="label" xsi:type="string" translate="true">Product Alerts</item>
                        <item name="collapsible" xsi:type="boolean">true</item>
                        <item name="opened" xsi:type="boolean">false</item>
                        <item name="sortOrder" xsi:type="string">2</item>
                        <item name="canShow" xsi:type="boolean">true</item>
                        <item name="componentType" xsi:type="string">fieldset</item>
                    </argument>
                </arguments>
            </block>
        </referenceBlock>
```

**Comment by Andrii-Deineha** [comment]
@chiragp-wagento Internal ticket MAGETWO-70086 was created.

**Comment by magento-engcom-team** [comment]
Internal ticket MAGETWO-70086

**Comment by raumatbel** [comment]
@okorshenko can you assign it me, please?

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-81966

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-81968

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-81967

**Comment by okorshenko** [comment]
Hi @chiragp-wagento the issue has been fixed in 2.1-develop and 2.2-develop branch. Will be available with upcoming patch release soon

**Comment by Ctucker9233** [comment]
@okorshenko Any idea what version of 2.1 will have it?
